vault/token_store.go

```diff
@@ -321,7 +341,7 @@ func NewTokenStore(c *Core, config *logical.BackendConfig) (*TokenStore, error)
 				Fields: map[string]*framework.FieldSchema{
 					"token": &framework.FieldSchema{
```

Comment:
The reason it's there is that the code later queries `FieldData` for that value, and if it's not a declared part of the schema it actually panics. So it's here to satisfy that need, but the function ensures that the value is set to the request's client token value before it's processed.